react/src/lib/List/examples/Default.js

```diff
@@ -20,3 +23,12 @@ export default class ListItemDefault extends React.PureComponent {
     );
   }
 }
+
+ListItemDefault.propTypes = {
```

Comment:
Default should not have anything added to it.

Reply:
Reverting `Default` to old state.